lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/components/RuntimeVersionManager.scala

```diff
@@ -401,7 +401,9 @@ class RuntimeVersionManager(
     */
   private def isEngineVersionCompatibleWithThisInstaller(
     manifest: Manifest
-  ): Boolean = CurrentVersion.version >= manifest.minimumRequiredVersion
+  ): Boolean =
+    CurrentVersion.isDevVersion ||
+    CurrentVersion.version >= manifest.minimumRequiredVersion
```

Comment:
```suggestion
  ): Boolean =
    if (CurrentVersion.version >= manifest.minimumRequiredVersion) true
    else if (CurrentVersion.isDevVersion) {
      logger.warn("The minimum required version for the selected engine is [${manifest.minimumRequiredVersion}]. The version check is ignored, however, because the current version [${CurrentVersion.version}] is a development version.
      true
    } else false
    
```

I think we should add a warning to the logs, as I feel like this can bite us in the future if we introduce some incompatible changes.